urdf_to_simox_xml/src/urdf_to_simox_xml.cpp

```diff
@@ -23,7 +23,6 @@
 #include <string>
 #include <sstream>
 #include <vector>
-#include <boost/foreach.hpp>
```

Comment:
I was getting the following errors:
[ 95%] Building CXX object shadow_robot_ethercat/sr_robot_lib/CMakeFiles/sr_hand_lib.dir/src/motor_updater.cpp.o
In file included from /home/beatriz/workspace/shadow/src/simox_ros/urdf_to_simox_xml/src/urdf_to_simox_xml.cpp:26:0:
/usr/include/boost/foreach.hpp:169:42: error: ‘boost::Q_FOREACH::tag’ has not been declared
 boost_foreach_is_lightweight_proxy(T *&, BOOST_FOREACH_TAG_DEFAULT) { return 0; }
                                          ^
/usr/include/boost/foreach.hpp:173:72: error: ‘boost::Q_FOREACH::tag’ has not been declared
 boost_foreach_is_lightweight_proxy(std::pair<T, T> *&, boost::foreach::tag) { return 0; }
                                                                        ^
/usr/include/boost/foreach.hpp:177:81: error: ‘boost::Q_FOREACH::tag’ has not been declared
 boost_foreach_is_lightweight_proxy(boost::iterator_range<T> *&, boost::foreach::tag) { return 0; }
                                                                                 ^
/usr/include/boost/foreach.hpp:181:76: error: ‘boost::Q_FOREACH::tag’ has not been declared
 boost_foreach_is_lightweight_proxy(boost::sub_range<T> *&, boost::foreach::tag) { return 0; }
                                                                            ^
/usr/include/boost/foreach.hpp:185:59: error: ‘boost::Q_FOREACH::tag’ has not been declared
 boost_foreach_is_lightweight_proxy(T **&, boost::foreach::tag) { return 0; }
                                                           ^
/usr/include/boost/foreach.hpp:194:36: error: ‘boost::Q_FOREACH::tag’ has not been declared
 boost_foreach_is_noncopyable(T *&, BOOST_FOREACH_TAG_DEFAULT) { return 0; }

I found that the boost libraries should be included before the Qt, which were included in the header. Moving it there before solved the issue. How do you suggest me to do it instead?